<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlockVideoBitrate</class>
 <widget class="QWidget" name="BlockVideoBitrate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="horizontalSpacing">
    <number>3</number>
   </property>
   <property name="verticalSpacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QCheckBox" name="videoBitrateButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Sets the average bitrate of the video,</string>
     </property>
     <property name="text">
      <string>Target bitrate</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="bitrateWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="AutoSelectDoubleSpinBox" name="videoBitRateEdit">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> Auto</string>
        </property>
        <property name="maximum">
         <double>300.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="DataComboBox" name="bitrateTypeBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="videoQualityWidget" native="true">
     <layout class="QHBoxLayout" name="qualityLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="losslessButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Automatically adjust the bitrate of the video according to a target quality.</string>
        </property>
        <property name="text">
         <string>Lossless</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="speedWidget" native="true">
     <layout class="QVBoxLayout" name="speedLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="speedButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Speed vs Quality. A higher value means a faster encoding but a lower video quality.</string>
     </property>
     <property name="text">
      <string>Speed</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="DataComboBox" name="tuneBox"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="tuneLabel">
     <property name="text">
      <string>Fine tune</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="videoQualityButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Automatically adjust the bitrate of the video according to a target quality.</string>
     </property>
     <property name="text">
      <string>Quality</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="intraButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Sets the average bitrate of the video,</string>
     </property>
     <property name="text">
      <string>Intra-frame</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionPerfect_95">
   <property name="text">
    <string>Perfect ( 95% )</string>
   </property>
  </action>
  <action name="actionAuto">
   <property name="text">
    <string>Auto</string>
   </property>
  </action>
  <action name="actionGood_60">
   <property name="text">
    <string>Good ( 60% )</string>
   </property>
  </action>
  <action name="actionProxy_30">
   <property name="text">
    <string>Proxy ( 30% )</string>
   </property>
  </action>
  <action name="actionBad_20">
   <property name="text">
    <string>Bad ( 20% )</string>
   </property>
  </action>
  <action name="actionBlu_Ray">
   <property name="text">
    <string>Blu-Ray ( 25 Mbps average )</string>
   </property>
  </action>
  <action name="actionDVD">
   <property name="text">
    <string>DVD ( 7Mbps average )</string>
   </property>
  </action>
  <action name="actionStreaming_12_Mbps">
   <property name="text">
    <string>Streaming ( 12 Mbps )</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataComboBox</class>
   <extends>QComboBox</extends>
   <header>UI/datacombobox.h</header>
  </customwidget>
  <customwidget>
   <class>AutoSelectDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>duqf-widgets/autoselectdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
